docs/references/basic-transactions.md

```diff
@@ -267,16 +267,31 @@ manually `MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr`.
 
 </SolanaCodeGroup>
 
-## How to change compute budget for a transaction
+## How to change compute budget, fee, &amp; priority for a transaction
+Transaction (TX) priority is transaction fee divided by compute units (CU), e.g. 
+fee/CU. By default the compute budget is 1.4M CU and the Base Fee is 5,000 
+Lamports. A microLamport is 0.000001 Lamports.
 
-Compute budget for a single transaction can be changed by adding an instruction
-call to the Compute Budget Program. By default the compute budget is set the product 
-of 200k compute units * number of instructions, with a max of 1.4M compute units. 
-The less compute you use, the less the transaction costs.
+The total compute budget or fee for a single TX can be changed by adding 
+instructions from the ComputeBudgetProgram. 
 
-**Note**: To change the compute budget for a transaction, you must make the 
-one of the first three instructions of the transaction the instruction that 
-sets the budget.
+Use `ComputeBudgetProgram.setComputeUnitLimit({ units: number })` to set
+the new compute budget. The value provided will replace the default value. 
+Transactions should request the minimum amount of CU required for 
+execution to maximize throughput, minimize fees, or increase priority.
```

Comment:
remove `, or increase priority`, because priority is solely determined by compute-unit-price. Reducing requested cu does reduce overall transaction fee tho.